app/src/routes/onboarding/forms/user.ts

```diff
@@ -0,0 +1,254 @@
+import { Field } from '@directus/types';
+import { useI18n } from 'vue-i18n';
+
+export const useUserFields = function (): Field[] {
+	const { t } = useI18n();
+	return [
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.first_name'),
+			field: 'first_name',
+			type: 'string',
+			schema: {
+				name: 'first_name',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
+				comment: null,
+				numeric_precision: null,
+				numeric_scale: null,
+				is_unique: false,
+				is_primary_key: false,
+				has_auto_increment: false,
+				foreign_key_schema: null,
+				foreign_key_table: null,
+				foreign_key_column: null,
+			},
+			meta: {
+				id: 17,
+				collection: 'onboarding',
+				field: 'first_name',
+				special: null,
+				interface: 'input',
+				options: { placeholder: t('fields.directus_users.first_name'), trim: true },
+				display: null,
+				display_options: null,
+				readonly: false,
+				hidden: false,
+				sort: 2,
+				width: 'half',
+				translations: null,
+				note: null,
+				conditions: null,
+				required: false,
+				group: null,
+				validation: null,
+				validation_message: null,
+			},
+		},
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.last_name'),
+			field: 'last_name',
+			type: 'string',
+			schema: {
+				name: 'last_name',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
```

Comment:
```suggestion
				max_length: 50,
```
The database has a max length of 50 here